c/src/sbp.c

```diff
@@ -372,15 +372,8 @@ s8 sbp_process(sbp_state_t *s, u32 (*read)(u8 *buff, u32 n, void *context))
       if (s->crc == crc) {
 
         /* Message complete, process it. */
-        s8 ret = SBP_OK_CALLBACK_UNDEFINED;
-        sbp_msg_callbacks_node_t* node;
-        for (node = s->sbp_msg_callbacks_head; node; node = node->next) {
-          if (node->msg_type == s->msg_type) {
-            (*node->cb)(s->sender_id, s->msg_len, s->msg_buff, node->context);
-            ret = SBP_OK_CALLBACK_EXECUTED;
-          }
-        }
-        return ret;
+        return sbp_process_payload(s, s->sender_id, s->msg_type, s->msg_len,
```

Comment:
just a nit - I'd assign the return value of `sbp_process_payload` to `ret` and continue to return `ret`. Easier for the debugger to inspect the value returned, etc.